### PR TITLE
Big update

### PR DESCRIPTION
**Pull request by Eddyskingdom**
add new sprite and new emote
(more work still comming in)

### CODE REVIEW

**Comment by Eddyskingdom** [comment]
why ant a allowd to make a big update to the game i have been stay onlie all night working on bugs and things that are use full later on and now two

**Comment by Harcourt1** [comment]
im asking why the dme changed

File: code/modules/WW2/_mags.dm [hunk 1]
```diff
@@ -24,6 +25,7 @@ WW 2 WEAPONS MAGS N AMMO
 
 /obj/item/ammo_magazine/mosinbox
 	name = "magazine box (7.62x54mm)"
+	desc = "this is a box with 7,62c54mm that is used for Mosin clips to get refilld"
 	icon_state = "mosin_ammo"
 //	origin_tech = "combat=2"
```
Comment (mattroks101) [inline code/modules/WW2/_mags.dm]: You guys don't use `origin_tech` anymore this should honestly just be deleted.

Comment (Eddyskingdom) [reply code/modules/WW2/_mags.dm]: ter is no reason to remove it any way just gives us extra work

**Comment by Eddyskingdom** [comment]
> im asking why the dme changed @Harcourt1 
i have not even messt with that the game just sometime add something by it self when i'm testing it

**Comment by Harcourt1** [comment]
Send me screenshots of the new sprites

File: code/modules/WW2/_mags.dm (hunk 1 above)
Comment (mattroks101) [reply code/modules/WW2/_mags.dm]: >deleting a few commented out lines is extra work

👌

Comment (Eddyskingdom) [reply code/modules/WW2/_mags.dm]: good that you understand

Comment (ShinHKPDes) [reply code/modules/WW2/_mags.dm]: absolute state of leben dev team

**Comment by Eddyskingdom** [comment]
@Harcourt1 still waiting

**Comment by cariosecus** [comment]
You need to fiz the spelling

**Pull request merged**